airflow/contrib/operators/ecs_operator.py

```diff
@@ -19,26 +19,12 @@
 
 import warnings
 
-from airflow.providers.amazon.aws.operators.ecs import ECSOperator, ECSProtocol as NewECSProtocol  # noqa
-from airflow.typing_compat import Protocol, runtime_checkable
+from airflow.providers.amazon.aws.operators.ecs import ECSOperator, ECSProtocol
```

Comment:
mm this classes are also deprecated.
it was renamed to `EcsOperator` and `EcsProtocol`

Reply:
Yeah, and I’m intentionally importing those deprecated ones so if someone imports this module (and misses the import-time deprecation warning), instantiating either class would also spit out a deprecation warning.

Reply:
Is  `pytest -s tests/core/test_core_to_contrib.py` passing? (I'm not sure if the test is executed as it's a core test)

I wonder if it causes troubles as we have a deprecated class pointing to a deprecated class
https://github.com/apache/airflow/blob/61b29d7454f024efce63a2cb684eb876ec7e4856/tests/deprecated_classes.py#L1067-L1068

Reply:
It passes. We currently only check the old import is deprecated, whether the new import is deprecated or _not_ is irrelevant.

Reply:
Shall we merge to make the PRs green again ?